### DISCUSSION TITLE
OpenAI以7萬小時電玩影片訓練AI玩Minecraft

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#865] [General]
沒有想過竟然能以影像訓練來大幅減少人力來讓AI自動的玩Minecraft

註：這可是我童年在玩的遊戲呀！如果可以用AI來玩那我豈不是可以短時間內就可以蓋完一棟建築，就不用像之前一樣花好幾天才蓋完一棟建築

https://www.ithome.com.tw/news/154487

**Comment by dabbie109303501** [comment]
雖然是很酷沒錯啦...但這樣不就少了自己玩遊戲的樂趣了嗎?

**Comment by a45320614** [comment]
Minecraft好玩的地方就是親手自己蓋建築阿阿
如果連玩遊戲都是AI來操控
那我乾脆不要玩好了qq

**Comment by dinosaur1009** [comment]
AI的效率驚人 但我可能還是偏向自己玩xdd

**Comment by Chiway1229** [comment]
從以前的2D下棋，沒想到這麼快就進展到3D遊戲上的應用了
說不定之後都能看到Ai在路上訓練模型了

**Comment by Pietra1226** [comment]
其實這個模型讓我們知道AI訓練已經強到無法想像了，希望能夠玩到更有沉浸式體驗的遊戲。

**Comment by ching020721** [comment]
AI蓋建築很厲害，但Minecraft就是享受花時間蓋出來成品的那種成就感

**Comment by yuxinlin93** [comment]
用ai蓋的話，如果馬上有想法，可以趁靈感正熱，把想像的建築蓋出來，感覺很有趣

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
這樣感覺跟開外掛沒啥差別 也少了自己玩的遊戲體驗

**Comment by williamyen0623** [comment]
這剛好可以解決想法源源不斷但是沒有時間和人力實踐的問題，雖然少了自己玩的樂趣，但卻能讓給一部分人更有效率的創作

**Comment by manipulateed** [comment]
我覺得其實很多東西，就是要人自己去嘗試去體驗，才能真正感受到他帶給我們的樂趣，所以並不是所有都適合ai 。

**Comment by HuangHsinTzu** [comment]
這可能可以當作當成課金誘因(?)
(對那些沒空研究但又想創造自己的世界的人可能有吸引力)
AI的學習力真的太強了
但我一直在想有一天AI會不會學會人類的情感
那真的會很可怕欸

**Comment by ruthchen1020** [comment]
AI在這方面真的強，但是它還是沒辦法擁有人類真正的創意。
希望人類可以善用AI卻不忘記自己存在的價值。

**Comment by evanchou1204** [comment]
剛好造福我這種連自己的豪宅都懶得蓋的懶人 好想叫AI幫我蓋房子 肯定又大又美

**Comment by Evan0979** [comment]
AI連玩個遊戲都要比我強嗚嗚嗚

**Comment by StevenWanglolz** [comment]
我有一個大膽的想法，如果訓練模型來speedrun MInecraft或是其他遊戲，只要餵足夠多的資料是不是有一天可以打破現在人類的紀錄?

**Comment by CPCpyt** [comment]
我覺得這樣的研究甚至能對未來機器人在日常生活行動有很深刻的啟發。
